orchagent/mirrororch.cpp

```diff
@@ -260,6 +265,76 @@ bool MirrorOrch::decreaseRefCount(const string& name)
     return true;
 }
 
+bool MirrorOrch::validateDstPort(const string& dstPort)
+{
+    Port port;
+    if (!m_portsOrch->getPort(dstPort, port) && port.m_type != Port::PHY)
+    {
+        SWSS_LOG_ERROR("Failed to locate port %s", dstPort.c_str());
```

Comment:
I think the log message should also specify if type is not `phy`. Can you add that to log and print type?

Reply:
Done. Added log